manifest_staging/deployment/provider-azure-installer.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   labels:
     app: csi-secrets-store-provider-azure
   name: csi-secrets-store-provider-azure
+  namespace: kube-system
```

Comment:
This is a breaking change. If the intention is to deploy the pods in `kube-system` then just `kubectl apply -n kube-system -f <file>` should be sufficient as the default namespace isn't set in the yamls.